ocaml/xapi-storage-script/main.ml

```diff
@@ -217,8 +227,8 @@ end = struct
       old PVS scripts *)
   let compat_uri device_config =
     match !V.version with
-    | Some version when String.(version = pvs_version) -> (
-      match List.Assoc.find ~equal:String.equal device_config "uri" with
```

Comment:
I think Base has this too, so we could reduce the changes needed here by replacing `open Core` with `open Base` at the top. (Unless at some point we'd want to remove the dependency on Base, but I see no reason to do that, reusing code is better than reinventing the wheel). But this is fine too, since you've already done it.